assets/js/timetree.js

```diff
@@ -326,15 +331,25 @@ function TreeGraph({ nodes, links, centuries }) {
         return link.value; // link strength defined when links created
       })
     )
-    // .force("link", d3.forceLink(links).distance(30).strength(link => {
-    // return 1;
-    // }))
     .force(
       "y",
       d3
         .forceY()
         .y((node) => centuryY(node))
         .strength(forceStrength.centuryY)
+    )
+    .force(
+      "x",
+      d3
+        .forceX()
+        .x((node) => branchX(node))
+        .strength((node) => {
```

Comment:
So this is a wholly new x-axis force for the branches.

Reply:
yes